README.md

```diff
@@ -40,13 +40,15 @@ ReplicaCAD             |  ScanNet
 ![Alt Text](.github/replicaCAD.gif)  |  ![Alt Text](.github/scanNet.gif)
 
 
-## Two iSDF Modes 
+## Three iSDF Modes 
 
-There're two options for running iSDF, chose the desired mode and then follow **only** instructions in that section:
+There are three options for running iSDF, chose the desired mode and then follow **only** instructions in that section:
 
 1. **[Running iSDF on sequences with pose provided](#1-running-isdf-on-sequences-with-pose-provided)**: for running iSDF on ScanNet and ReplicaCAD sequences used in the paper. 
 2. **[Running iSDF with a live camera in ROS](#2-running-isdf-with-a-live-camera-in-ros)**: a ros wrapper for iSDF subscribes to a topic of posed frames that is published by an ORB-SLAM3 wrapper.
+3. **[Running iSDF with a Franka + live camera in ROS](#3-running-isdf-with-a-franka-+-live-camera-in-ros)**: a ros wrapper for iSDF subscribes to a topic of realsense frames and poses from a calibrated realsense on a Franka Panda robot.
```

Comment:
This link wasn't working for me, I think you need to remove the +

Reply:
Fixed with new commit, thanks!